doc/en/components/radial-gauge.md

```diff
@@ -335,9 +335,15 @@ The scale is visual element that highlights full range of values in the gauge wh
 `sample="/gauges/radial-gauge/scale", height="320", alt="{Platform} radial gauge scale"`
 
 
-## Labels
+## Labels and Titles
 The radial gauge labels are visual elements displaying numeric values at a specified interval between values of the `MinimumValue` and `MaximumValue` properties. You can position labels by setting the `LabelExtent` property to a fraction, where 0 represents center of gauge and 1 represents outer extent of the gauge backing. Also, you can customize labels setting various styling properties such as `FontBrush` and `Font`.
 
+`TitleText` and `SubtitleText` properties are available and can both be used to display custom text for the needle. In addition, `TitleDisplaysValue` and `SubtitleDisplaysValue` will let you replace the custom text with the needle's value. So you can occupy custom text for the title but show the value via the subtitle and vice versa.
+
+If the highlight needle is shown, as explained below, then custom text can be shown via  `HighlightLabelText`, otherwise `HighlightLabelDisplaysValue` can be enabled and display it's value.
+
+Each of these labels for the needle have various styling attributes you can apply to change the font, angle, brush and distance from the center of the gauge such as `TitleExtent`, `TitleAngle`, `SubtitleFontSize`, `HighlightLabelBrush`. 
+
```

Comment:
this should be in a new section called Gauge Title and Subtitle and add code snippets that just sets some props for Title and Subtitle feature